plugins/outputs/http/README.md

```diff
@@ -0,0 +1,34 @@
+# HTTP Output Plugin
+
+This plugin writes to a HTTP Server using the `POST Method`.
+
+Data collected from telegraf is sent in the Request Body.
+
+### Configuration:
+
+```toml
+# Send telegraf metrics to HTTP Server(s)
+[[outputs.http]]
+  ## It requires a url name.
+  ## Will be transmitted telegraf metrics to the HTTP Server using the below URL.
+  ## Note that not support the HTTPS.
+  url = "http://127.0.0.1:8080/metric"
+  ## http_headers option can add a custom header to the request.
+  ## The value is written as a delimiter(:).
+  ## Content-Type is required http header in http plugin.
+  ## so content-type of HTTP specification (plain/text, application/json, etc...) must be filled out.
+  http_headers = [ "Content-Type:application/json" ]
+  ## With this HTTP status code, the http plugin checks that the HTTP request is completed normally.
+  ## As a result, any status code that is not a specified status code is considered to be an error condition and processed.
+  expected_status_codes = [ 200, 204 ]
+  ## Configure response header timeout in seconds. Default : 3
+  response_header_timeout = 3
+  ## Configure dial timeout in seconds. Default : 3
+  dial_timeout = 3
```

Comment:
Let's start with only a timeout that covers the entire request (`http.Client.Timeout`).  I'm not sure we need timeouts that cover only parts of the request but we can add them in later if needed.

Reply:
I got it. I remained only timeout configuration.